src/Calculator/Views/CalculatorStandardOperators.xaml

```diff
@@ -350,45 +233,28 @@
                 <ColumnDefinition x:Name="FnC2" Width="1*"/>
                 <ColumnDefinition x:Name="FnC3" Width="1*"/>
             </Grid.ColumnDefinitions>
-            <controls:CalculatorButton x:Name="PercentButton"
-                                       x:Uid="percentButton"
-                                       Grid.Column="0"
-                                       Style="{StaticResource SymbolOperatorButtonStyle}"
-                                       AutomationProperties.AutomationId="percentButton"
-                                       ButtonId="Percent"
-                                       Content="&#xE94C;"/>
-            <controls:CalculatorButton x:Name="SquareRootButton"
-                                       x:Uid="squareRootButton"
-                                       Grid.Column="1"
+            <controls:CalculatorButton x:Name="InvertButton"
+                                       x:Uid="invertButton"
                                        Style="{StaticResource SymbolOperatorButtonStyle}"
-                                       AutomationProperties.AutomationId="squareRootButton"
-                                       ButtonId="Sqrt"
-                                       Content="&#xF899;"/>
+                                       FontSize="18"
```

Comment:
>FontSize="18" [](start = 39, length = 13)

Should this always be 18?

Reply:
These get overwritten by the visual sates. Same for other comments regarding fonts.